source/lib/src/gpu/tabulate.cu

```diff
@@ -195,10 +249,18 @@ __global__ void tabulate_fusion_se_a_grad_fifth_order_polynomial(
     const int last_layer_size,
     const bool is_sorted) {
   bool enable_se_atten = two_embed != nullptr;
+#if GOOGLE_CUDA
   extern __shared__ int _data[];
+  int MTILE_OR_KTILE = MTILE;
+#elif TENSORFLOW_USE_ROCM
+  HIP_DYNAMIC_SHARED(int, _data)
+  int MTILE_OR_KTILE = KTILE;
```

Comment:
I must confirm if `KTILE` -> `MTILE` by #830 should also apply to ROCm.

Reply:
Yes, they both have the same value 4, defined in the header of the file.
```
#define MM 4
#define KK 4
```